src/abstractdataframe/show.jl

```diff
@@ -251,6 +256,7 @@ required for printing have been precomputed.
   required to render each column.
 - `leftcol::Int`: The index of the first column in a chunk to be rendered.
 - `rightcol::Int`: The index of the last column in a chunk to be rendered.
+- `rowid`: Used internally to handle showing `DataFrameRow`
```

Comment:
Everything is used internally here anyway. :-)

Also missing ending period.

Reply:
Fixed (initially I exposed it, and later decided to hide the internals altogether)